src/style.css

```diff
@@ -0,0 +1,103 @@
+* {
+  box-sizing: border-box;
+  font-family: Arial, serif;
```

Comment:
Don't set the font using * selector. You can set it for the whole page/body and inherit where needed